app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,50 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
generally, app-specific css code should go in something like main.css.  application.css should just be a toplevel file that includes other stuff.  pls move this css to its own file.